lib/pure/math.nim

```diff
@@ -158,6 +158,23 @@ proc classify*(x: float): FloatClass =
     return fcSubnormal
   return fcNormal
 
+proc approxEqual*[T: SomeFloat](x, y: T; ulp: Natural = 4): bool {.noSideEffect.} =
```

Comment:
Use ``unitsInLastPlace`` or ``relevantDigits`` or something better instead of ``ulp``.